tox.ini

```diff
@@ -36,6 +36,7 @@ description =
     run tests
     alldeps: with all optional dependencies
     devdeps: with the latest developer version of key dependencies
+    romandeps: with dependencies specific to supporting Roman
```

Comment:
should the dev version of roman datamodels be added to devdeps as well?

Reply:
Good question. Does it break a lot in dev? I know some pipeline merge their PRs precariously, resulting in broken dev version a lot. How does this package handle their merges?

Reply:
I ask that because if it breaks a lot, it is pointless to test against their dev. But if not, then sure, we can.